service/src/test/java/uk/gov/hmcts/reform/fpl/service/docmosis/NoticeOfHearingGenerationServiceTest.java

```diff
@@ -85,17 +90,21 @@ void shouldBuildExpectedTemplateDataWithHearingDateAndTimeWhenHearingStartAndEnd
 
         CaseData caseData = buildCaseData(now.toLocalDate());
         HearingBooking hearingBooking = buildHearingBooking(now, now);
-        final DocmosisNoticeOfHearing docmosisNoticeOfHearing = service.getTemplateData(caseData, hearingBooking);
+        final DocmosisNoticeOfHearing actualDocmosisNoticeOfHearing = service.getTemplateData(caseData, hearingBooking);
         DocmosisNoticeOfHearing.DocmosisNoticeOfHearingBuilder docmosisNoticeOfHearingBuilder
             = getDocmosisNoticeOfHearingBuilder(now.toLocalDate());
 
-        docmosisNoticeOfHearingBuilder.hearingDate(formatLocalDateTime(now, DATE));
-        docmosisNoticeOfHearingBuilder.preHearingAttendance(formatLocalDateTime(now.minusHours(1), "h:mma"));
-        docmosisNoticeOfHearingBuilder.hearingTime(String.format("%s - %s",
-            formatLocalDateTime(now, HEARING_TIME_FORMAT),
-            formatLocalDateTime(now, HEARING_TIME_FORMAT)));
+        docmosisNoticeOfHearingBuilder.hearingBooking(getDocmosisHearingBookingBuilder()
```

Comment:
you can do chaining for all of this:

`        DocmosisNoticeOfHearing expectedDocmosisNoticeOfHearing = getDocmosisNoticeOfHearingBuilder(now.toLocalDate())
            .hearingBooking(getDocmosisHearingBookingBuilder()
                .hearingTime(String.format("%s - %s",
                    formatLocalDateTime(hearingBooking.getStartDate(), HEARING_DATE_AND_TIME_FORMAT),
                    formatLocalDateTime(hearingBooking.getEndDate(), HEARING_DATE_AND_TIME_FORMAT)))
                .preHearingAttendance(formatLocalDateTime(now.minusHours(1), DATE_TIME))
                .hearingDate("")
                .build())
            .build();`

Reply:
Yep let's clean that up.